thinc/model.py

```diff
@@ -562,7 +562,7 @@ def from_dict(self, msg: Dict) -> "Model":
             for param_name, value in msg["params"][i].items():
                 if value is not None:
                     value = node.ops.asarray(value)
-                node.set_param(param_name, value)
+                node.set_param(param_name, value.copy())
```

Comment:
```suggestion
                node.set_param(param_name, value)
```

Reply:
hehe yea, we just pushed the same commit ;-)